scalableminds/webknossos
# Improve activativate-user flow

### URL of deployed dev instance (used for testing):
- https://improveuseractivation.webknossos.xyz

### Steps to test:
- go to users page
- deactivate a user
- activate a user
- this prompt should show up:
![image](https://github.com/scalableminds/webknossos/assets/2486553/96509346-b374-448c-8f65-b9500f074dec)
- ok/cancel should both work as expected

### Issues:
- fixes #7219
